src/nbl/video/COpenGLCommandBuffer.h

```diff
@@ -575,26 +575,43 @@ class COpenGLCommandBuffer final : public IGPUCommandBuffer
         return true;
     }
 
-    bool drawMeshBuffer(const nbl::video::IGPUMeshBuffer* meshBuffer) override
+    inline bool drawMeshBuffer(const nbl::video::IGPUMeshBuffer* meshBuffer) override
     {
         const auto* pipeline = meshBuffer->getPipeline();
         const auto bindingFlags = pipeline->getVertexInputParams().enabledBindingFlags;
+        auto vertexBufferBindings = meshBuffer->getVertexBufferBindings();
 
         for (uint16_t i = 0; i < nbl::asset::SVertexInputParams::MAX_ATTR_BUF_BINDING_COUNT; ++i)
         {
-            // TODO
+            if (bindingFlags & core::createBitmask({ i }))
```

Comment:
not like this, bind everything at once.

Reply:
what if the bindings are not placed tightly? I mean that each binding isn't in sequence? Can I just bind all `nbl::asset::SVertexInputParams::MAX_ATTR_BUF_BINDING_COUNT` and pass `meshBuffer->getVertexBufferBindings()`?

Reply:
theres no requirement on the used binding slots to be consecutive, any unused binding should be nullptr anyway

Binding using the whole array is how its always been done (because its only one bind call).

Its important that the null bindings be "bound" because that unbinds the previous binding at that position.

Reply:
oh, I understand and never thought about how it was done before, thank you:)

Reply:
corrected in following commit